package.json

```diff
@@ -156,6 +150,11 @@
       "import": "./handlers/shared/index.mjs",
       "require": "./handlers/shared/index.js"
     },
+    "./handlers/actions": {
+      "types": "./handlers/actions/index.d.ts",
+      "import": "./handlers/actions/index.mjs",
+      "require": "./handlers/actions/index.js"
+    },
```

Comment:
what is the use case of this being exported? Why App would need that?

Reply:
I think on 1.x better to *not* export more than we need. Every time we export something we must be sure it should be there, because its public api and everyone can build on top

Unless someone asks for, I'd prefer to export only platform specific logic, while we control the internals. The we can evolve into exposing this

Reply:
All right, I haven't though about that this way 😅 

I'll remove internals export then 👍🏻